### ISSUE TITLE
Add configuration options (width, height) for video artifacts

### ISSUE THREAD

**Issue by sergiomap** [wix/Detox#1451]
On Android the video recording is empty if the screen resolution is set to 1440x2960. If I set the resolution to 1080x1920 the video is fine.

**Comment by noomorph** [comment]
Hello, @sergiomap!
Thanks for the report.

Could you manually experiment with various CLI parameters here and report what worked out for you?
http://adbshell.com/commands/adb-shell-screenrecord

**Comment by LeoNatan** [comment]
Could this be a limitation of the recording system?

**Comment by noomorph** [comment]
It's a third party issue (depends on emulator, adb shell screenrecord, etc), I am just trying to figure out how can Detox help to alleviate that.

**Comment by sergiomap** [comment]
I tried to run:
`adb shell screenrecord sdcard/demo2.mp4 --size 1440x2960`

and got the error:

Unable to get output buffers (err=-38)
Encoder failed (err=-38)

**Comment by noomorph** [comment]
@sergiomap, please try other resolutions with the same aspect ratio (18:37), it is interesting to know if a smaller resolution below FullHD is going to work out.

**Comment by sergiomap** [comment]
Nop, didn't worked.
I just managed put it working with 16:9.

**Comment by sergiomap** [comment]
Do you have some way to record the video in a different resolution of the emulator using Detox? I mean, change the adb shell record or using other parameter.

**Comment by noomorph** [comment]
@sergiomap, no, we don't have other ways at the moment.

Related to #1403
Subtask of #1238

**Comment by noomorph** [comment]
Resolved via #1923.

Try out this syntax in `package.json` Detox section:

```json
{
  "detox": {
    "artifacts": {
      "plugins": {
        "video": {
          "android": {
            "size": [1080, 1920],
          }
        }
      }
    },
  }
}
```
